active_learning_single_annotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,639 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Active Learning with Autogluon\n",
+    "Train accurate models with minimal data labeling via Active Learning, and AutoML"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/examples/blob/master/active_learning_single_annotator/active_learning_single_annotator.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates a practical approach to active learning for training an AutoGluon model with cleanlab. \n",
+    "\n",
+    "In **Active Learning**, we aim to construct a labeled dataset by collecting the fewest labels that still allow us to train an accurate classifier model. Here we assume data labeling is done in **batches**, and between these data labeling rounds, we retrain our classifier to decide what previously unlabeled examples (i.e. datapoints) to label next round.\n",
+    "\n",
+    "**AutoGluon** is an automated machine learning library used to efficiently train a model while **Cleanlab** provides an active learning score quantifying how desirable it is to collect an additional label for every possible unlabeled example\n",
```

Comment:
```suggestion
```

Reply:
don't think these need explanation, want to keep text as concise as possible in these notebooks